src/tabpfn/classifier.py

```diff
@@ -689,6 +696,7 @@ def _raw_predict(self, X: XType, *, return_logits: bool) -> torch.Tensor:
 
         return self.forward(X, use_inference_mode=True, return_logits=return_logits)
 
+    @track_model_call(model_method="predict", param_names=["X"])
```

Comment:
As this calls `predict_proba().` we will end up with two logs. I guess this can be filtered out when looking at the data, but would it make analysis easier to create a `_predict_proba()` function and call this from both `predict()` and `predict_proba()`, thus avoiding the double log?